sysbuild/Kconfig.netcore

```diff
@@ -39,10 +39,10 @@ config NRF_DEFAULT_MULTIPROTOCOL
 
 choice NETCORE
 	prompt "Netcore image"
-	default NETCORE_EMPTY if NRF_DEFAULT_EMPTY && ($(BOARD) = "nrf5340dk_nrf5340_cpuapp" || $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns" || $(BOARD) = "thingy53_nrf5340_cpuapp" || $(BOARD) = "thingy53_nrf5340_cpuapp_ns")
-	default NETCORE_HCI_RPMSG if NRF_DEFAULT_BLUETOOTH && ($(BOARD) = "nrf5340dk_nrf5340_cpuapp" || $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns" || $(BOARD) = "thingy53_nrf5340_cpuapp" || $(BOARD) = "thingy53_nrf5340_cpuapp_ns")
-	default NETCORE_802154_RPMSG if NRF_DEFAULT_802154 && ($(BOARD) = "nrf5340dk_nrf5340_cpuapp" || $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns" || $(BOARD) = "thingy53_nrf5340_cpuapp" || $(BOARD) = "thingy53_nrf5340_cpuapp_ns")
-	default NETCORE_MULTIPROTOCOL_RPMSG if NRF_DEFAULT_MULTIPROTOCOL && ($(BOARD) = "nrf5340dk_nrf5340_cpuapp" || $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns" || $(BOARD) = "thingy53_nrf5340_cpuapp" || $(BOARD) = "thingy53_nrf5340_cpuapp_ns")
+	default NETCORE_EMPTY if NRF_DEFAULT_EMPTY && ($(BOARD) = "nrf5340dk_nrf5340_cpuapp" || $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns" || $(BOARD) = "thingy53_nrf5340_cpuapp" || $(BOARD) = "thingy53_nrf5340_cpuapp_ns" || $(BOARD) = "nrf7002dk_nrf5340_cpuapp" || $(BOARD) = "nrf7002dk_nrf5340_cpuapp_ns")
```

Comment:
This is minor, but I think it's not the best approach to copy+paste the same board names for every default option. Considering the fact we already have all of them listed in `config SUPPORT_NETCORE` and additionally every `NRF_DEFAULT_X` depends on some specific boards, wouldn't be enough to just add `if SUPPORT_NETCORE` before the `choice NETCORE`?

Reply:
I see. @nordicjm, @tejlmand can we do what Kamil suggested? Or maybe something like the following would work?

choice NETCORE
	prompt "Netcore image"
	default NETCORE_EMPTY if NRF_DEFAULT_EMPTY && SUPPORT_NETCORE
	default NETCORE_HCI_RPMSG if NRF_DEFAULT_BLUETOOTH && SUPPORT_NETCORE
	default NETCORE_802154_RPMSG if NRF_DEFAULT_802154 && SUPPORT_NETCORE
	default NETCORE_MULTIPROTOCOL_RPMSG if NRF_DEFAULT_MULTIPROTOCOL && SUPPORT_NETCORE
	default NETCORE_NONE
	depends on SUPPORT_NETCORE